app/views/delivery_partners/participants/index.html.erb

```diff
@@ -1,3 +1,23 @@
 <h1 class="govuk-heading-xl">Participants</h1>
 
-<p>Hello world</p>
+<table class="govuk-table">
```

Comment:
there is a table dsl available which might work better, see https://govuk-components.netlify.app/components/table/ from https://github.com/DFE-Digital/govuk-components